trefoil/cli/crs.py

```diff
@@ -0,0 +1,37 @@
+import click
+from netCDF4 import Dataset
+from pyproj import Proj
+
+from trefoil.netcdf import crs
+from trefoil.cli import cli
+
+
+@cli.command(short_help='Set spatial reference information for a variable in an existing dataset')
+@click.argument('filename', type=click.Path(exists=True))
+@click.argument('proj4')
+@click.argument('variables', required=False)
+@click.option('--all', 'all_variables', is_flag=True, default=False, help='Set CRS for all variables in the dataset')
```

Comment:
Is there really a case where we wouldn't want to set the CRS for all data variables in the file?  At least in how we tend to use them, there aren't multiple geospatial projections in the same file.  

I think `--all` should be the default and probably could even be removed as a flag.  Instead, I'd flip this around and provide a `--only` followed by a list of variables for those rare cases where you only want it applied to specific variables.

Reply:
The `set_crs` API seems to suggest you can set CRS per variable (as it takes a single variable name to set the CRS for).